<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateLocalVersioRepoDialog</class>
 <widget class="QDialog" name="CreateLocalVersioRepoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>142</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Versio Repository</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleBox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>User name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="usernameBox"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
